ui/raidboss/data/03-hw/dungeon/fractal_continuum.txt

```diff
@@ -0,0 +1,156 @@
+hideall "--reset--"
+hideall "--sync--"
+hideall "--start--"
+
+0 "--reset--" sync /:Exhibit level III is no longer sealed/ window 10000 jump 0
+0 "--reset--" sync /:The high-level incubation bay is no longer sealed/ window 10000 jump 0
+0 "--reset--" sync /:The reality augmentation bay is no longer sealed/ window 10000 jump 0
+
+
+###Phantom Ray
+#-ii F7D
+
+0 "Start"
+0.0 "--start--" sync /Exhibit level III will be sealed off/ window 0,1
+10.3 "Rapid Sever" sync /:Phantom Ray:F7A:/
+16.6 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+21.4 "Double Sever 1" sync /:Phantom Ray:F7B:/
+25.6 "Double Sever 2" sync /:Phantom Ray:F7C:/
+27.4 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+
+41.5 "Rapid Sever" sync /:Phantom Ray:F7A:/
+47.7 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+52.5 "Double Sever 1" sync /:Phantom Ray:F7B:/
+56.5 "Double Sever 2" sync /:Phantom Ray:F7C:/
+58.3 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/ window 8,10 jump 27.4
+
+72.4 "Rapid Sever" sync /:Phantom Ray:F7A:/ jump 41.5
+78.6 "Atmospheric Displacement"
+83.3 "Double Sever 1"
+87.4 "Double Sever 2"
+89.2 "Atmospheric Displacement"
+#Phase 2 at < 75% HP
+
+97.7 "--sync--" #landing for phase 2 loop
+100.0 "Damage Up" sync /:Phantom Ray:F7F:/  window 100,30 #Overclock
+104.8 "Double Sever 1" sync /:Phantom Ray:F7B:/
+107.6 "Atmospheric Compression" sync /:Phantom Ray:F80:/
+109.0 "Double Sever 2" sync /:Phantom Ray:F7C:/
+111.8 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+118.9 "Rapid Sever" sync /:Phantom Ray:F7A:/
+124.0 "Rapid Sever" sync /:Phantom Ray:F7A:/
+128.2 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+133.1 "Double Sever 1" sync /:Phantom Ray:F7B:/
+137.3 "Double Sever 2" sync /:Phantom Ray:F7C:/
+139.0 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
+146.1 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/ jump 97.7
+
+148.4 "Damage Up" sync /:Phantom Ray:F7F:/ jump 100.0
```

Comment:
I wish this was automated (or maybe timelines should just have a loop keyword??), but can you make sure the times in your loop lookahead match the times before? If you run `python util/timeline_adjust.py --file=ui/raidboss/data/03-hw/dungeon/fractal_continuum.txt --adjust=48.4`, then you'll get this output:

```
148.4 "Damage Up" sync /:Phantom Ray:F7F:/  window 100,30 #Overclock
153.2 "Double Sever 1" sync /:Phantom Ray:F7B:/
156.0 "Atmospheric Compression" sync /:Phantom Ray:F80:/
157.4 "Double Sever 2" sync /:Phantom Ray:F7C:/
160.2 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
167.3 "Rapid Sever" sync /:Phantom Ray:F7A:/
172.4 "Rapid Sever" sync /:Phantom Ray:F7A:/
176.6 "Atmospheric Displacement" sync /:Phantom Ray:F7E:/
```

...which doesn't quite line up with these times.  (48.4 is because you're jumping from 148.4 back to 100.0).  If the times line up perfectly, then when the loop happens, the timeline won't jump perceptively.  Sorry this is a pain.  <_<

This is what I was trying to get at in https://github.com/quisquous/cactbot/blob/master/docs/TimelineGuide.md#building-loops

Reply:
That's an artifact, I think. It looks like on this one I had run through a long encounter with Phantom Ray and simply adjusted the second phase once for both loops and tossed it in. I suppose I should verify the lookaheads for Minotaur and Curator as well.

Reply:
It looks like I handled the Minotaur/Curator loops "correctly" in that sense, so I'll just fix the Phantom Ray lookahead and that should take care of things.